p8e-util/src/main/kotlin/io/p8e/crypto/SignerImpl.kt

```diff
@@ -0,0 +1,22 @@
+package io.p8e.crypto
+
+import com.google.protobuf.Message
+import io.p8e.proto.Common.Signature
+import java.security.KeyPair
+
+open class SignerImpl {
```

Comment:
Considering implementing the Pen class in here and label the SignerImpl class as the default, but wanted to separate both the signers for now so it easier to visualize and test.